Sources/AlgoliaSearchClient/Models/Search/Response/SearchResponse/Auxiliary/Facet/AttributedFacets.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  AttributedFacets.swift
+//  
+//
+//  Created by Vladislav Fitc on 17/03/2021.
+//
+
+import Foundation
+
+public struct AttributedFacets: Codable {
```

Comment:
Should we moved this class to InstantSearch since the client doesn't use it?

Reply:
wouldn't people not using InstantSearch still need to use it?

Reply:
Probably yes, however, to maintain the separation of concerns, I don't think the API client should be providing it.
You think we should do anyway?

Reply:
Good point, I moved it to the [InstantSearch PR](https://github.com/algolia/instantsearch-ios/pull/168). 
This structure is the output of the `FacetOrderer` algorithm which is only available in the InstantSearch.